sgv2-filter.meta.js

```diff
@@ -3,10 +3,11 @@
 // @namespace   https://github.com/GarionCZ/sgv2-filter
 // @description Giveaway filter for SteamGifts v2
 // @author      Garion
-// @include     http://www.steamgifts.com/*
+// @include     http://www.steamgifts.com/giveaways*
```

Comment:
I don't think this will work on the mainpage since without SG++ the main page is http://www.steamgifts.com, SG++ forwards it to http://www.steamgifts.com/giveaways to make things easier. This plugin should work without SG++ as well though.